react-app/components/ListImages/components/ImageItem/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext, useState, memo } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+import PropTypes from 'prop-types';
+import { getItemLocalStorage, setItemLocalStorage } from '@utils/localStorage';
+import { GalleryContext } from '@contexts/galleryContext';
+
+import './styles.css';
+
+function ImageItem(props) {
+  const { id: idImageItem, url, name, size } = props;
+  const [isHovered, setIsHovered] = useState(false);
+  const { setIsGalleryOpened, setFilesToUploadList } = useContext(GalleryContext);
+
+  const handleMouseEnter = () => {
+    setIsHovered(true);
+  };
+
+  const handleMouseLeave = () => {
+    setIsHovered(false);
+  };
+
+  const handleOpenGallery = () => {
+    setIsGalleryOpened(true);
+  };
+
+  const handleDeleteFile = () => {
+    const filesToUploadListStorage = getItemLocalStorage('filesToUploadList') || [];
+    const filesToUploadListUpdated = filesToUploadListStorage.filter(({ id }) => id !== idImageItem);
+    setItemLocalStorage('filesToUploadList', filesToUploadListUpdated);
+    setFilesToUploadList(filesToUploadListUpdated);
+  };
```

Comment:
Can we add docstrings to all these handlers?

Reply:
And docstrings for the modules too

Reply:
Why for the modules? I can do it for handlers